README.rst

```diff
@@ -154,7 +154,7 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
```

Comment:
Based on the number of commits to the repository, I did not change the order of authors.
```
$ git shortlog -sne
  280  Dongdong Tian <seisman.info@gmail.com>
   256  Leonardo Uieda <leouieda@gmail.com>
   143  Wei Ji <23487320+weiji14@users.noreply.github.com>
    86  Will Schlitzer <schlitzer90@gmail.com>
    42  Meghan Jones <meghanj@hawaii.edu> + <meghanj@alum.mit.edu>
    42  Michael Grund <23025878+michaelgrund@users.noreply.github.com> + <michael_grund@gmx.de>
    31  Yao Jiayuan <coreman.seism@gmail.com>
    15  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  kmaterna <kathrynmaterna@gmail.com>
     2  Abhishek Anant <81291713+itsabhianant@users.noreply.github.com>
     2  Tyler Newton <54011699+tjnewton@users.noreply.github.com>
    1  Malte Ziebarth <malte.ziebarth@fmvkb.de>
```